ethereum/README.md

```diff
@@ -2,61 +2,101 @@
 
 This directory contains smart contracts utilized by the Polkadot-Ethereum Bridge.
 
+- Bridge.sol: application registry and routing of messages from Substrate
+- Application.sol: an abstract contract that must be implemented by any Bridge application
+    - ETHApp.sol: application for cross-chain ETH transfers between Ethereum and Substrate
+    - ERC20App.sol: application for cross-chain ERC20 transfers between Ethereum and Substrate
+- Decoder.sol: a library for decoding SCALE encoded data
+- Verifier.sol: verifies tx origin and signatures
+- Scale.sol: implements decoding of SCALE encoded compact uints; not currently used, will support future work on generalized data relay
+
 ## Set up
 
-Create a `.env` file using the following template:
+After starting the blockchain and deploying the contracts, the Bridge's Ethereum component is ready for usage. To generate configuration information required for configuring the Parachain and Relayer components, run the set up scripts described in scripts section below.
 
 ```bash
-cp .env.example .env
+# Install dependencies
+yarn install
 ```
 
-Install dependencies
+Start truffle environment with a local Ethereum network and deploy the contracts
 
 ```bash
-yarn install
+truffle develop
```

Comment:
This could be confusing for users since `truffle develop` doesn't exit.

Try this rather:

Open a fresh terminal window and start a local truffle environment:
```bash
truffle develop
```

Migrate contracts:
```bash
truffle migrate --all
```

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/107/commits/5d02b9d9f0e7f00a7a8082af1a854a096165dee6